### PR TITLE
Add a documentation for FxCop

### PR DESCRIPTION
**Pull request by gnakagaw**
This PR adds a document for [FxCop](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers), a new static analyzer in Sider.
See: https://github.com/sider/runners/pull/731

### CODE REVIEW

File: docs/tools/csharp/fxcop_analyzers.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+id: fxcop_analyzers
+title: FxCop Analyzers
+sidebar_label: FxCop Analyzers
+hide_title: true
+---
+# FxCop Analyzers
+> This is **BETA**. The behavior of this tool might change,
```
Comment (ybiquitous) [inline docs/tools/csharp/fxcop_analyzers.md]: Can you please replace it with "This is not published yet." like GolangCI-Lint?

https://github.com/sider/sider-docs/blob/1bf5609eb0017c2fef3195cd3d1d9aa727816dea/docs/tools/go/golangci-lint.md

File: docs/tools/csharp/fxcop.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+> This is **not published yet**.
+
+| Supported Version | Language             | Website                  |
+| :---------------- | :------------------- | :----------------------- |
+| 2.9.8             | C#                   | [dotnet/roslyn-analyzers](https://github.com/dotnet/roslyn-analyzers#microsoftcodeanalysisfxcopanalyzers) |
+
+**FxCop** is a static analysis tool for [.NET  platform](https://dotnet.microsoft.com/). It supports both of C# and Visual Basic .NET. However, Sider supports C# project only now.
+
+## Limitations for beta version
+This is a beta release. We have several limitation for analysis.
```
Comment (yykamei) [inline docs/tools/csharp/fxcop.md]:
```suggestion
This is a beta release. We have several limitations for analysis.
```

File: docs/tools/csharp/fxcop.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+* We support only projects with .NET Core.
+  * No support for mono, Unity, .NET Framework, and so on.
+* We support only projects using [MSBuild](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild) as its build management system.
+  * When you use Visual Studio to develop, it's ok in almost cases.
+  * No support for other build system.(e.g. [Cake](https://cakebuild.net/), [Nuke](https://nuke.build/), GNU Make, etc..)
```
Comment (yykamei) [inline docs/tools/csharp/fxcop.md]:
```suggestion
  * No support for other build systems (e.g. [Cake](https://cakebuild.net/), [Nuke](https://nuke.build/), GNU Make, etc..) 
```

File: docs/tools/csharp/fxcop.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+**FxCop** is a static analysis tool for [.NET  platform](https://dotnet.microsoft.com/). It supports both of C# and Visual Basic .NET. However, Sider supports C# project only now.
+
+## Limitations for beta version
+This is a beta release. We have several limitation for analysis.
+* We support only projects with .NET Core.
+  * No support for mono, Unity, .NET Framework, and so on.
+* We support only projects using [MSBuild](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild) as its build management system.
+  * When you use Visual Studio to develop, it's ok in almost cases.
+  * No support for other build system.(e.g. [Cake](https://cakebuild.net/), [Nuke](https://nuke.build/), GNU Make, etc..) 
+* We ignore custom ruleset (*.ruleset) file.
```
Comment (yykamei) [inline docs/tools/csharp/fxcop.md]:
```suggestion
* We ignore custom ruleset (`*.ruleset`) file.
```

**Comment by gnakagaw** [comment]
@ybiquitous @yykamei Thanks for kind review comments. I've fixed the issues. Please squash and merge or rebase and merge when you merge this. This PR contains many WIP and fix commits.

File: docs/tools/csharp/fxcop.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+## Limitations for beta version
+
+This is a beta release. We have several limitations for analysis.
+
+- We support only projects with .NET Core.
+  - No support for mono, Unity, .NET Framework, and so on.
+- We support only projects using [MSBuild](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild) as its build management system.
+  - When you use Visual Studio to develop, it's ok in almost cases.
+  - No support for other build systems.(e.g. [Cake](https://cakebuild.net/), [Nuke](https://nuke.build/), GNU Make, etc..)
+- We ignore custom ruleset (`\*.ruleset`) file.
```
Comment (yykamei) [inline docs/tools/csharp/fxcop.md]: This backslash looks unnecessary.

```suggestion
- We ignore custom ruleset (`*.ruleset`) file.
```

https://deploy-preview-308--sider-docs.netlify.com/tools/csharp/fxcop
Screenshot from 2020-03-02 19-10-17

File: docs/tools/csharp/fxcop.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+## Getting Started
+
+To start using FxCop, enable it in [Repository Settings](../../getting-started/repository-settings.md).
+
+If you put your project configration file (\*.csproj) not in project root dir, you have to set `root_dir` parameter in `sider.yml` file.
```
Comment (yykamei) [inline docs/tools/csharp/fxcop.md]:
```suggestion
If you put your project configration file (`*.csproj`) not in project root dir, you have to set `root_dir` parameter in `sider.yml` file.
```

**Comment by gnakagaw** [comment]
Thanks.

**Pull request merged**